File: MAP/lab4/src/view/RunExample.java
package view;

import controller.Controller;
import exceptions.DivisionByZero;
import exceptions.EmptyStack;
import exceptions.InvalidOperator;
import exceptions.VariableNotDefined;
import exceptions.fileExceptions.FileException;

public class RunExample extends Command {
    private Controller ctrl;

    public RunExample(String key, String desc, Controller ctr){
        super(key, desc);
        this.ctrl=ctr;
    }
    @Override
    public void execute() {
        try {
            ctrl.executeAll();
        }
        catch (EmptyStack eStk){
            System.out.println(eStk);
        }
        catch(DivisionByZero div){
            System.out.println(div);
        }
        catch(InvalidOperator invO){
            System.out.println(invO);
        }
        catch(VariableNotDefined varND){
            System.out.println(varND);
        }
        catch(FileException f){
            System.out.println(f);
        }

    }
}

File: MAP/lab7/lab7/View/RunExample.cs
using System;

namespace lab7.View
{
   
    public class RunExample : Command {
    private Controller.Controller ctrl;
    public RunExample(string key, string desc, Controller.Controller ctr) : base(key, desc)
    {
        this.ctrl=ctr;
    }
    
    public override void execute() {

        try
        {
            ctrl.executeAll();
        }
        catch (SystemException e)
        {
            Console.WriteLine(e.Message);
        }
    }
        
    }
}